deploy/install.ps1

```diff
@@ -122,6 +122,13 @@ if ($Version -eq "edge") {
     $downloadURL = "ghcr.io/${GitHubOrg}/rad/${OS}-${Arch}:latest"
     Write-Output "Downloading edge CLI from ${downloadURL}..."
     oras pull $downloadURL -o $RadiusRoot
+
+    # Check if the oras pull command was successful
+    if ($LASTEXITCODE -ne 0) {
+        Write-Output "Failed to download edge rad CLI."
+        Write-Output "Please visit https://edge.docs.radapp.io/installation for edge build installation instructions."
```

Comment:
Would a user have a way to identify that it failed due to expired token? Should it be included in the docs linked?

Reply:
Oras unfortunately gives a bad `403 Unauthorized` error without much additional context on an expired PAT, even when the targeted registry is public. I talked to the oras maintainers about this and they said it was intentional. But agreed we can add some docs to help with this.

Reply:
Updated, thanks for the suggestion!